src/app/home/directives/home-story.component.ts

```diff
@@ -95,4 +65,30 @@ export class HomeStoryComponent implements OnInit {
     }
   }
 
+  get isPlusSign(): boolean {
+    return this.story.isNew && !this.story.isStored()
```

Comment:
That works. Thanks. Now I just need to figure out what to do if they set Explicit/Clean and nothing else, or maybe why Explicit/Clean isn't registering as changed...